Sprint-3/1-implement-and-rewrite-tests/implement/2-is-proper-fraction.js

```
@@ -8,9 +8,13 @@
 // write one test at a time, and make it pass, build your solution up methodically
 
 function isProperFraction(numerator, denominator) {
-  if (numerator < denominator) {
+  if(typeof(numerator) != "number" || typeof(denominator) != "number") {
+    return "[ wrong input ]";
+  }
+  if (numerator < denominator ) {
     return true;
   }
+  return false;
 }
```

In mathematics, -A/B == A/-B == -(A/B), and -A/-B == A/B for any integers A and B (B ≠ 0).
They represent a proper fraction if A < B and B ≠ 0.

So `isProperFraction(-4, 3)` should return `false` because 4 >= 3.

So `isProperFraction(-2, 5)` should return `true` because 2 < 5.

Consider comparing the absolute value of the numerator and the denominator instead.


In addition, to indicate errors, it is best practices to ***throw an error*** (instead of returning an error message).
If you have time, you should check out how to throw an error in JS.

Alternatively, you can choose to return `false` to indicate something is not a fraction.